wdl-lint/RULES.md

```diff
@@ -6,35 +6,35 @@ be out of sync with released packages.
-| `MissingMetas`                   | Completeness, Clarity         | Ensures that tasks have both a meta and a parameter_meta section.                     |
-| `MissingOutput`                  | Completeness, Portability     | Ensures that tasks have an output section.                                            |
-| `NoCurlyCommands`                | Clarity                       | Ensures that tasks use heredoc syntax in command sections.                            |
-| `PascalCase`                     | Clarity, Naming, Style        | Ensures that structs are defined with PascalCase names.                               |
-| `PreambleComments`               | Clarity, Spacing, Style       | Ensures that documents have correct comments in the preamble.                         |
-| `PreambleWhitespace`             | Spacing, Style                | Ensures that documents have correct whitespace in the preamble.                       |
-| `RuntimeSectionKeys`             | Completeness, Deprecated      | Ensures that runtime sections have the appropriate keys.                              |
-| `SectionOrdering`                | Sorting, Style                | Ensures that sections within tasks and workflows are sorted.                          |
-| `SnakeCase`                      | Clarity, Naming, Style        | Ensures that tasks, workflows, and variables are defined with snake_case names.       |
-| `Todo`                           | Completeness                  | Ensures that `TODO` statements are flagged for followup.                              |
-| `TrailingComma`                  | Style                         | Ensures that lists and objects in meta have a trailing comma.                         |
-| `Whitespace`                     | Spacing, Style                | Ensures that a document does not contain undesired whitespace.                        |
-
+| Name                             | Tags                          | Description                                                                                       |
+| :------------------------------- | :---------------------------- | :------------------------------------------------------------------------------------------------ |
+| `CallInputSpacing`               | Style, Clarity, Spacing       | Ensures proper spacing for call inputs                                                            |
+| `CommandSectionMixedIndentation` | Clarity, Correctness, Spacing | Ensures that lines within a command do not mix spaces and tabs.                                   |
+| `CommentWhitespace`              | Spacing                       | Ensures that comments are properly spaced.                                                        |
+| `ContainerValue`                 | Clarity, Portability          | Ensures that the value for `container` keys in `runtime`/`requirements` sections are well-formed. |
```

Comment:
The only thing that changed it this line—everything else is just a spacing change for the last column.